csharp/ql/src/Likely Bugs/ThreadUnsafeICryptoTransformLambda.ql

```diff
@@ -0,0 +1,96 @@
+/**
+ * @name Potential usage of a n object implementing ICryptoTransform class in a way that would be unsafe for concurrent threads.
```

Comment:
There is still a space between `a` and `n` here.

Reply:
Fixed it for real this time